cmd/gapit/flags.go

```diff
@@ -175,7 +179,7 @@ type (
 		Out   string        `help:"the file to generate"`
 		Local struct {
 			Port       int       `help:"capture a local program instead of using ADB"`
-			App        file.Path `help:"a local program to trace"`
+			App        string    `help:"a local program to trace"`
```

Comment:
Given that you've changed `file.Path` to `string`, does that mean it might not be local? If so, update help text?

Reply:
I had planned on renaming -local-app to "-desktop-app" and updating all of the surrounding text. I will do that.